### PR TITLE
Correct path of default log4j configures

### PR DESCRIPTION
**Pull request by goodseeyou**
This PR is to correct default configure path for log4j in helm chart.

### CODE REVIEW

**Comment by goodseeyou** [comment]
Any instruction to add labels? If it's necessary. Thanks.

File: helm/pinot/values.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ controller:
 
   jvmOpts: "-XX:ActiveProcessorCount=2 -Xms256M -Xmx1G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xlog:gc*:file=/opt/pinot/gc-pinot-controller.log"
 
-  log4j2ConfFile: /opt/pinot/etc/config/pinot-controller-log4j2.xml
+  log4j2ConfFile: /opt/pinot/etc/conf/pinot-controller-log4j2.xml
```
Comment (walterddr) [inline helm/pinot/values.yaml:115]: which configuration stated it should be `conf` instead of `config` ?

Comment (walterddr) [reply helm/pinot/values.yaml:115]: nvm i think it is from https://github.com/apache/pinot/blob/99d34e05e0fbe287df9fdabe1237b78459ae3e93/docker/images/pinot/Dockerfile#L45

**Review by walterddr — COMMENTED** [review]
lgtm tagging @xiangfu0 to take another look. how can this work for so long :-P

File: helm/pinot/values.yaml (hunk 1 above)
Comment (goodseeyou) [reply helm/pinot/values.yaml:115]: here is config located in image I found
https://github.com/apache/pinot/tree/master/docker/images/pinot/etc/conf

Comment (goodseeyou) [reply helm/pinot/values.yaml:115]: I guess the configuration for conf path is from here 
https://github.com/apache/pinot/blob/master/pinot-distribution/pinot-assembly.xml#L243

**Review by Jackie-Jiang — APPROVED** [review]

**Pull request merged**